component.json

```diff
@@ -1,6 +1,6 @@
 {
```

Comment:
We should just drop this file. I forgot to delete it when I merged the two repos.